resources/language/resource.language.ko_kr/strings.po

```diff
@@ -314,8 +322,644 @@ msgstr "새로 고칠 때 몇분 마다 확인할까요?"
+msgstr "포트"
+
+msgctxt "#30205"
+msgid "Test database connection"
+msgstr "데이타베이스 시험접속"
+
+msgctxt "#30206"
+msgid "ERROR: The MySQL database is not reachable - the local database will be used"
+msgstr "ERROR: MySQL 데이터베이스와 연결되지 않아, 로컬 데이타베이스를 사용합니다."
+
+msgctxt "#30207"
+msgid "Set this device as main auto-updates manager"
+msgstr "이 장치를 자동 업데이트 매니저로 사용합니다."
+
+msgctxt "#30208"
+msgid "Check if this device is the main auto-update manager"
+msgstr "이 장치가 주 매니저인지 확인합니다."
+
+msgctxt "#30209"
+msgid "This device now handles the auto-updates of shared-library"
+msgstr "이 장치가 공유라이브러리의 자동 업데이트를 지금 합니다."
+
+msgctxt "#30210"
+msgid "This device handles the auto-updates of shared-library"
+msgstr "이 장치가 공유라이브러리의 자동 업데이트를 합니다."
+
+msgctxt "#30211"
+msgid "Another device handles the auto-updates of shared-library"
+msgstr "다른 장치가 공유라이브러리의 자동 업데이트를 합니다."
+
+msgctxt "#30212"
+msgid "There is no device that handles the auto-updates of shared-library"
+msgstr "공유라이브러리를 자동업데이트할 장치가 없습니다."
 
 msgctxt "#30213"
 msgid "InputStream Helper settings..."
 msgstr "InputStream Helper 애드온 설정"
+
+msgctxt "#30214"
+msgid "Force the update of my list"
+msgstr "찜한 것들을 강제 업데이트"
+
+msgctxt "#30215"
+msgid "Highlight titles included in my list with a color"
+msgstr "찜한 것에 있는 타이틀을 색으로 강조"
+
+msgctxt "#30216"
+msgid "Show ESN in use"
+msgstr "사용 중인 ENS확인"
+
+msgctxt "#30217"
+msgid "Reset ESN"
+msgstr "ESN 재설정"
+
+msgctxt "#30218"
+msgid "Are you sure you want to reset the ESN?\r\nIf you confirm, wait until you receive the login message."
+msgstr "ESN재설정하시렵니까?\r\n확인후, 로그인화면이 나타날 때까지 기다리십시오."
+
+msgctxt "#30219"
+msgid "Disable notification for synchronization completed"
+msgstr "동기화완료 알림을 사용하지 않습니다."
+
+msgctxt "#30220"
+msgid "The Kodi library has been updated"
+msgstr "Kodi라이브러리를 새로 고쳤습니다."
+
+msgctxt "#30221"
+msgid "Owner account"
+msgstr "주인장"
+
+msgctxt "#30222"
+msgid "Kid account"
+msgstr "키즈"
+
+msgctxt "#30223"
+msgid "Synchronization now uses \"My List\" of profile {}"
+msgstr "프로파일 {}의 \"찜한 것들\"을 동기화에 사용합니다."
+
+msgctxt "#30224"
+msgid "Auto update mode"
+msgstr "자동으로 고치기"
+
+msgctxt "#30225"
+msgid "Manual"
+msgstr "수동"
+
+msgctxt "#30226"
+msgid "Scheduled"
+msgstr "자동"
+
+msgctxt "#30227"
+msgid "Synchronize Kodi library with My List"
+msgstr "Kodi라이브러리와 내가 찜한 것들을 동기화"
+
+msgctxt "#30228"
+msgid "Use \"My list\" of the current profile"
+msgstr "사용 중인 프로파일로부터 \"찜한 것들\"을 가져옵니다."
+
+msgctxt "#30229"
+msgid "Show the name of the selected profile"
+msgstr "프로파일 이름을 보여줍니다."
+
+msgctxt "#30230"
+msgid "Check for updates now"
+msgstr "업데이트를 지금 확인합니다"
+
+msgctxt "#30231"
+msgid "Do you want to check update now?"
+msgstr "업데이트를 확인하시렵니까?"
+
+msgctxt "#30232"
+msgid "Restrict by Maturity Level"
+msgstr "연령제한"
+
+msgctxt "#30233"
+msgid "Content for {} will require the PIN to start playback."
+msgstr "{}을(를) 보려면 PIN을 입력하십시오."
+
+msgctxt "#30234"
+msgid "Keep watched status marks separate for each profile (this option will be removed in future versions)"
+msgstr "    "
```

Comment:
forgot?

Reply:
Probably I forgot to commit my last change 😉